index.d.ts

```diff
@@ -5729,7 +5737,7 @@ declare type CAPI = {
    *
    * See https://www.sqlite.org/c3ref/vfs_find.html
    */
-  sqlite3_vfs_find: (vfsName: string) => sqlite3_vfs;
+  sqlite3_vfs_find: (vfsName: string|null) => WasmPointer;
```

Comment:
`find` doesn't return the vfs directly, needs a separate `new sqlite_vfs(ptr)` call